presto-native-execution/presto_cpp/presto_protocol/java-to-struct-json.py

```diff
@@ -170,7 +170,7 @@ def member_name(name):
 def special(filepath, current_class, key, classes, depends):
     classes[current_class].class_name = current_class
     (status, stdout, stderr) = classes[current_class][key] = util.run(
-        "../../velox/scripts/license-header.py --header ../../velox/license.header --remove "
+        "../../velox/scripts/license-header.py --header ../../license.header --remove "
```

Comment:
The license.header is slightly different, using the correct one for presto_cpp allows it to be removed. As far as I can tell, this script is only called for presto protocol files, so the velox license.header should not be used.